config/samples/nfs_rc.yaml

```diff
@@ -0,0 +1,34 @@
+
+apiVersion: v1
```

Comment:
Whether need the file

Reply:
create for nfs server need it:
at first
 `kubectl apply -f nfs_rc.yaml`
then
 `kubectl apply -f nfs_server.yaml`
nfs server create success